app/platform/fabric/sync/SyncPlatform.ts

```diff
@@ -86,6 +88,8 @@ export class SyncPlatform {
 			this.network_name = args[1];
 		}
 
+		this.blockCount = network_configs[this.network_id].blockCount;
```

Comment:
blockCount needs to be defined in config.json in (app/platform/fabric/config.json)